.github/workflows/main.yml

```
@@ -0,0 +1,48 @@
+name: CI/CD Setup
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+      - continuous-integration-setup
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest pytest-cov Django
```

You can install from your requirements.txt files like so:

```suggestion
          pip install -r requirements/base.txt -r requirements/test.txt 
```